website/archiver/tasks.py

```diff
@@ -201,6 +201,8 @@ def archive_addon(addon_short_name, job_pk, stat_result):
     addon_name = addon_short_name
     if 'dataverse' in addon_short_name:
         addon_name = 'dataverse'
+        revision = 'latest' if addon_short_name.split('-')[-1] == 'draft' else 'latest-published'
```

Comment:
Cool. Could you make an improvement ticket in jira (with the component Maintenance) with you thoughts on a potential refactor?